lib/features/digital_guide_view/general_info/presentation/widgets/report_change_button.dart

```diff
@@ -35,3 +40,28 @@ class ReportChangeButton extends StatelessWidget {
     );
   }
 }
+
```

Comment:
Function here could be replaced or greatly simplified by following extension that is already defined in our codebase

```dart
extension LaunchUrlUtilX on WidgetRef? {
  Future<bool> launch(String uriStr) async {
    if (this != null && uriStr.startsWith(UrlConfig.topwrUrl)) {
      final urlWithoutDomain = uriStr.split(UrlConfig.topwrUrl).last;
      unawaited(
        this!.navigateNamedUri(urlWithoutDomain),
      );
      return true;
    }
    final uri = Uri.parse(uriStr);
    if (await canLaunchUrl(uri)) {
      return launchUrl(uri);
    }
    return false;
  }
}
```